gp-includes/router.php

```diff
@@ -59,97 +59,97 @@ private function default_routes() {
 		$dir      = '([^_/][^/]*)';
 		$path     = '(.+?)';
 		$projects = 'projects';
-		$project  = $projects.'/'.$path;
+		$project  = $projects . '/' . $path;
 		$id       = '(\d+)';
 		$locale   = '(' . implode( '|', wp_list_pluck( GP_Locales::locales(), 'slug' ) ) . ')';
 		$set      = "$project/$locale/$dir";
 
 		// overall structure
 		return array(
-			'/'                                               => array( 'GP_Route_Index', 'index' ),
+			'/'                                            => array( 'GP_Route_Index', 'index' ),
 
-			'get:/profile'                                    => array( 'GP_Route_Profile', 'profile_view' ),
-			"get:/profile/$path"                              => array( 'GP_Route_Profile', 'profile_view' ),
+			'get:/profile'                                 => array( 'GP_Route_Profile', 'profile_view' ),
+			"get:/profile/$path"                           => array( 'GP_Route_Profile', 'profile_view' ),
 
-			'get:/settings'                                   => array( 'GP_Route_Settings', 'settings_get' ),
-			'post:/settings'                                  => array( 'GP_Route_Settings', 'settings_post' ),
+			'get:/settings'                                => array( 'GP_Route_Settings', 'settings_get' ),
+			'post:/settings'                               => array( 'GP_Route_Settings', 'settings_post' ),
 
-			"get:(/languages)/$locale/$dir/glossary"          => array( 'GP_Route_Glossary_Entry', 'glossary_entries_get' ),
-			"post:(/languages)/$locale/$dir/glossary"         => array( 'GP_Route_Glossary_Entry', 'glossary_entries_post' ),
-			"post:(/languages)/$locale/$dir/glossary/-new"    => array( 'GP_Route_Glossary_Entry', 'glossary_entry_add_post' ),
+			"get:(/languages)/$locale/$dir/glossary"       => array( 'GP_Route_Glossary_Entry', 'glossary_entries_get' ),
+			"post:(/languages)/$locale/$dir/glossary"      => array( 'GP_Route_Glossary_Entry', 'glossary_entries_post' ),
+			"post:(/languages)/$locale/$dir/glossary/-new" => array( 'GP_Route_Glossary_Entry', 'glossary_entry_add_post' ),
 			"post:(/languages)/$locale/$dir/glossary/-delete" => array( 'GP_Route_Glossary_Entry', 'glossary_entry_delete_post' ),
-			"get:(/languages)/$locale/$dir/glossary/-export"  => array( 'GP_Route_Glossary_Entry', 'export_glossary_entries_get' ),
-			"get:(/languages)/$locale/$dir/glossary/-import"  => array( 'GP_Route_Glossary_Entry', 'import_glossary_entries_get' ),
+			"get:(/languages)/$locale/$dir/glossary/-export" => array( 'GP_Route_Glossary_Entry', 'export_glossary_entries_get' ),
+			"get:(/languages)/$locale/$dir/glossary/-import" => array( 'GP_Route_Glossary_Entry', 'import_glossary_entries_get' ),
 			"post:(/languages)/$locale/$dir/glossary/-import" => array( 'GP_Route_Glossary_Entry', 'import_glossary_entries_post' ),
```

Comment:
This looks odd, see https://github.com/pedro-mendonca/GlotPress-WP/blob/1f29dc7138c98cb185e57ae7f4243f5cac915e24/gp-includes/router.php#L77-L83

Reply:
I wonder why phpcbf didn't format this one

Reply:
Even if I fix it, phpcbf undoes it.
I've changed the arrays to multiline, to see if it's a false positive somehow, it still changes the format to a bad alignemnt, of only one space.
I'll ignore it for now and align the whole array as it should.

Reply:
The issue here is also Arrays.MultipleStatementAlignment maxColumn.
Changed it to 80 and it's ok now.